packages/datatrak-web-server/src/routes/SurveyRoute.ts

```diff
@@ -116,6 +121,9 @@ export class SurveyRoute extends Route<SurveyRequest> {
             .sort((a: any, b: any) => a.componentNumber - b.componentNumber),
         };
       })
+      // Hide Task questions from the survey. They are not displayed in the web app and are
+      // just used to trigger new tasks in the TaskCreationHandler
```

Comment:
Theoretically we won't need to to this, since there is handling on import of survey questions that sets `hidden` to `true`, but no harm in doing this here too

Reply:
ah I didn't know that we were doing that but yeah probably no harm in doing it here too